### ISSUE TITLE
InvalidParameterException in aws_ecs_service

### ISSUE THREAD

**Issue by guidoffm** [hashicorp/terraform#23941]
Getting an exception with no further details:

```
module.terraform.aws_ecs_task_definition.service_definition: Creating...
module.terraform.aws_ecs_task_definition.service_definition: Creation complete after 0s [id=svc1-fargate-definition-1]
module.terraform.aws_ecs_service.ecs_service: Creating...

Error: InvalidParameterException:  "svc1-fargate"
```

**Comment by guidoffm** [comment]
Duplicate of #23920

**Comment by guidoffm** [comment]
Run with
`TF_LOG=trace terraform apply`
you see the cause of the Exception. At the moment the cause of the problem is swallowed by Terraform and it should be displayed.

**Comment by guidoffm** [comment]
In the trace log for  #23920 you see the cause too.

**Comment by guidoffm** [comment]
I get the exception cause displayed using this command:

`TF_LOG=trace terraform apply -auto-approve 2>&1 |grep aws-sdk-go |grep InvalidParameterException|grep message`

**Comment by bflad** [comment]
Hi @guidoffm 👋 Thank you for reporting this issue and sorry you are running into trouble. While you did not provide the information from the [bug report template](https://github.com/hashicorp/terraform/issues/new?template=bug_report.md), this appears very similar to a regression in version 2.45.0 of the Terraform AWS Provider (https://github.com/terraform-providers/terraform-provider-aws/issues/11682) and was resolved in [Terraform AWS Provider version 2.46.0](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#2460-january-23-2020), which was released January 23rd. Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading. 👍
